web/app/views/TableRow.java

```diff
@@ -34,44 +34,8 @@ public String process(JsonNode doc, String property, String param,
 			List<String> vs = values;
 			return vs.isEmpty() ? ""
 					: String.format("<tr><td>%s</td><td>%s</td></tr>", label,
-							vs.stream().map(val -> label(doc, property, param, val, keys))
-									.collect(Collectors.joining(
-											property.equals("subjectChain") ? " <br/> " : " | ")));
-		}
-
-		private String label(JsonNode doc, String property, String param,
-				String val, Optional<List<String>> labels) {
-			String value = property.equals("subjectChain")
-					? val.replaceAll("\\([\\d,]+\\)$", "").trim() : val;
-			if (!labels.isPresent()) {
-				return refAndLabel(property, value, labels)[0];
-			}
-			String term = value;
-			if (param.equals("q")) {
-				term = "\"" + value + "\"";
-			}
-			try {
-				term = URLEncoder.encode(term, "UTF-8");
-			} catch (UnsupportedEncodingException e) {
-				Logger.error("Could not call encode '{}'", term, e);
-			}
-			String search = String.format("/resources/search?%s=%s", param, term);
-			JsonNode node = doc.get(property);
-			String label = labelForId(value, node, labels);
-			String result = labels.get().contains("numbering") ? label
-					: String.format(
-							"<a title=\"Nach weiteren Titeln suchen\" href=\"%s\">%s</a>",
```

Comment:
Don't we lose these links with this change? Also a few lines below, "Linked-Data-Quelle abrufen". Is this deployed somewhere to see the UI?

Reply:
Now deployed to http://stage.lobid.org/resources/ .

Reply:
Alright, links are there (e.g. http://stage.lobid.org/resources/990067007460206441), wherever they are coming from.